app/src/main/java/com/framgia/englishconversation/screen/videoDetail/VideoDetailViewModel.java

```diff
@@ -103,4 +114,12 @@ private void releasePlayer() {
             mSimpleExoPlayer = null;
         }
     }
+
+    public void onCreateCommentTouched() {
+        releasePlayer();
+        Bundle bundle = new Bundle();
+        bundle.putParcelable(Constant.EXTRA_TIMELINE, mTimelineModel);
+        mNavigator.startActivityForResult(CreateCommentActivity.class, bundle,
```

Comment:
2.11 Argument in Activity and Fragment
When passing date into an Activity or Framgment by Intent or Bundle, keys must follow the rules 2.10 and need to declare public static.

In case of passing a user into activity, call getProfileIntent()

public static Intent getProfileIntent(Context context, User user) {
    Intent intent = new Intent(context, ProfileActivity.class);
    intent.putParcelableExtra(EXTRA_USER, user);
    return intent;
}
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md